### DISCUSSION TITLE
Remove Dependency on MQTT

### DISCUSSION THREAD

**Discussion by warwickbuilds** [blakeblackshear/frigate#1538] [Ideas]
Hi, would love to run this service but I've moved away from MQTT for my setup, I would like to possible see the dependency dropped for MQTT.

With the availability of the Home Assistant Integration, I assume this is using direct API integration and therefore MQTT is redundant for this configuration.

**Comment by blakeblackshear** [comment]
MQTT is not redundant. The integration still uses mqtt under the hood.

**Reply by warwickbuilds** [reply]
ah ok, just my assumption, which was obviously wrong :) 

Looks like a great solution 👏

**Comment by Artekus** [comment]
I utilize it in Node-Red and with HA so removing MQTT would be detrimental to my setup. I imagine there are others who utilize the MQTT apart from HA as well.

**Reply by warwickbuilds** [reply]
I didn't say remove, I certainly realise lots use it, but lots also don't and limits the usage of this service which I would love to use.

And with the direct HA integration now available I thought MQTT could become just 1 of the integration options, I now know from above that also just uses the MQTT dependency.

All good, just a future request that I feel would allow more people to easily pickup and use this
